packages/lesswrong/server/resolvers/petrovDay2024Resolvers.ts

```diff
@@ -0,0 +1,48 @@
+import { PetrovDayLaunchs } from "@/lib/collections/petrovDayLaunchs";
+import { addGraphQLMutation, addGraphQLQuery, addGraphQLResolvers, addGraphQLSchema } from "../vulcan-lib";
+import { petrovDayLaunchCode } from "@/components/seasonal/PetrovDayButton";
+import PetrovDayActions from "@/lib/collections/petrovDayActions/collection";
+import { petrovBeforeTime } from "@/components/Layout";
+import { DatabaseServerSetting } from "../databaseSettings";
+import sample from "lodash/sample";
+
+const petrovFalseAlarmMissileCount = new DatabaseServerSetting<number[]>('petrovFalseAlarmMissileCount', [])
+const petrovRealAttackMissileCount = new DatabaseServerSetting<number[]>('petrovRealAttackMissileCount', [])
+
+const getIncomingCount = (incoming: boolean) => {
+  if (incoming) {
+    return sample(petrovRealAttackMissileCount.get())
```

Comment:
does this not also depend on how many generals pressed the button, on the side that launched nukes?  or no?

Reply:
it does not